yarn.lock

```diff
@@ -6182,6 +6182,11 @@ url-parse-lax@^3.0.0:
   dependencies:
     prepend-http "^2.0.0"
 
+url-polyfill@^1.1.10:
```

Comment:
@jc00ke Did you regenerate the yarn lockfile? When I try to build locally yarn tells me the checkout needs to rebuild it, so I think it wasn't changed when you pinned this lib.

Reply:
Sorry about that. Forced push to fix.